github/SelfHostedActionsRunner.pyi

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Dict, List
+
+from github.GithubObject import NonCompletableGithubObject
+
+class SelfHostedActionsRunner(NonCompletableGithubObject):
+    def __repr__(self) -> str: ...
+    def _initAttributes(self) -> None: ...
+    def _useAttributes(self, attributes: Dict[str, Any]) -> None: ...
+    @property
+    def id(self) -> int: ...
+    @property
+    def name(self) -> str: ...
+    @property
+    def os(self) -> str: ...
+    @property
+    def status(self) -> str: ...
+    @property
+    def busy(self) -> bool: ...
+    def labels(self) -> List[Dict[str, Any]]: ...

```

Comment:
Are you sure the value is Any?

Reply:
The output sample was 
```
{
    "id": 5,
    "name": "self-hosted",
    "type": "read-only"
}
```

I changed it to `Union[str, int]` for now